### ISSUE TITLE
Add rudimentary camera selector

### ISSUE THREAD

**Issue by brettfiedler** [phetsims/paper-land#85]
We are having multiple people running into the issues of the tool picking the incorrect camera (sometimes ignoring browser preference). Anyone with a laptop and an external webcam will face this challenge. JG and I also had issues with OBS virtual cam taking precedence as well.

Let's add a really quick and rough selector for the cameras ASAP!

**Comment by jbphet** [comment]
I have something very basic working, and will attempt to build on this, but it might be enough to get us through the upcoming demo.  I've added the ability to specify an alternate camera using a query parameter called `useCamera`.  The value of the parameter reflects the order in which the system lists the cameras, so to specify an alternative to the default you'll generally want to use the value `useCamera=1`.  Also, you need to do this on both the camera and projector pages.  In most cases the URLs will look like:

- http://localhost:3000/camera.html?useCamera=1 and
- http://localhost:3000/projector.html?useCamera=1

**Comment by BLFiedler** [comment]
Made a quick change to the landing page to make the default links to Camera and Projector include this query. By default it should make a difference if you only have 1 camera. I added a warning to make sure to adjust both pages if selecting another camera.

![image](https://user-images.githubusercontent.com/47331395/235272509-01738c91-c7f8-47da-80a6-bc8e17f20196.png)

**Comment by BLFiedler** [comment]
Making this a control could be tricky if it needs to happen to both Camera and Projector (and they remain different pages). Maybe we could have the selector on the landing page to dynamically modify the links instead?

**Comment by jbphet** [comment]
I have a more sophisticated version of this working, but it's had minimal testing, so it's on a branch at the moment.  The branch is called `camera-selector`.  I'll test it a bit more tomorrow and will potentially move it to master based on how things look.

**Comment by jbphet** [comment]
This is implemented and working pretty well.  I have tested several use cases of dynamically plugging in and removing an external camera, and the code seemed to handle all of them reasonably well.  I have merged the branch where this code was developed back into master.

I would recommend the next steps of testing it out on a number of different systems and making sure it words properly on all of them, and also following up on https://github.com/phetsims/paper-land/issues/91.
